tuplex/CMakeLists.txt

```diff
@@ -139,6 +139,14 @@ if(GENERATE_PDFS)
 else()
     message(STATUS "PDF generation for ASTs and logical plans disabled. Enable by setting -DGENERATE_PDFS=ON.")
 endif()
+
+# suppress "#warning" directive otuput
+option(SHOW_EXPLICIT_WARNINGS "Show the output of #warning directives in the code (lots of output)" OFF)
+if(SHOW_EXPLICIT_WARNINGS)
```

Comment:
maybe add message(STATUS "Disabled output of #warning directives in the code, use SHOW_EXPLICIT_WARNINGS to turn all warnings on") or so?